applications/app/views/videoEmbed.scala.html

```diff
@@ -78,7 +78,7 @@
                                 || 'XDomainRequest' in window
                             )
                         ),
-                        config: @fragments.javaScriptConfig(model.SimpleContentPage(video))
+                        config: @Html(templates.js.javaScriptConfig(model.SimpleContentPage(video)).body)
```

Comment:
My understanding: because this file is a `.scala.html` file, and we don't want to escape the characters here, we wrap it in `Html` so that it does not get HTML encoded.

I tried `JavaScript` and here's what happened:

``` html
config: 


    {
        &quot;page&quot;:
```

… etc.